includes/admin/class-rop-admin.php

```diff
@@ -1572,17 +1572,26 @@ public function rop_get_yoast_seo_active_status() {
 	public function rop_get_wpml_languages() {
 
 		if ( $this->rop_get_wpml_active_status() === false ) {
-					 return;
+			return;
 		}
 
 		$wpml_active_languages = apply_filters( 'wpml_active_languages', null, array('skip_missing' => 1) );
 
+		// Get TranslatePress publish plugin languages.
+		if ( empty( $wpml_active_languages ) && class_exists( 'TRP_Translate_Press' ) ) {
```

Comment:
Since we already have a hook for languages, `'wpml_active_languages,'` you can move this to a separate function, then add it as `add_filter`. 

Bonus: the user will be able to modify the result via a separate filter if he wants.

Reply:
Done!